### PR TITLE
Removed self. when not needed

### PR DESCRIPTION
**Pull request by staffler-xyz**
Removed "self." calls where not needed. According to most code style guides (for example https://github.com/raywenderlich/swift-style-guide#use-of-self) they should be avoided where possible. What is your opinion about this?

I didn't add a changelog because it's only a style change (or should I?).

## Checklist

- [x ] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [x] New extensions are written in Swift 4.
- [x] New extensions support iOS 8.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+.
- [x] I have added tests for new extensions, and they passed.
- [x] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [x] All extensions are declared as **public**.
- [x] I have added a [changelog](https://github.com/SwifterSwift/SwifterSwift/blob/master/CHANGELOG_GUIDELINES.md) entry describing my changes.

### CODE REVIEW

File: Sources/Extensions/AppKit/NSViewExtensions.swift [hunk 1]
```diff
@@ -132,7 +132,7 @@ extension NSView {
     ///
     /// - Parameter subviews: array of subviews to add to self.
     public func addSubviews(_ subviews: [NSView]) {
-        subviews.forEach({self.addSubview($0)})
+        subviews.forEach({addSubview($0)})
```
Comment (LucianoPAlmeida) [inline Sources/Extensions/AppKit/NSViewExtensions.swift]: I know is not the purpose of the PR but can you also fix here to use trailing syntax and adding space between like `subviews.forEach { addSubview($0) }`? :))

File: Sources/Extensions/UIKit/UIViewExtensions.swift [hunk 2]
```diff
@@ -280,7 +280,7 @@ public extension UIView {
     ///
     /// - Parameter subviews: array of subviews to add to self.
     public func addSubviews(_ subviews: [UIView]) {
-        subviews.forEach({ self.addSubview($0) })
+        subviews.forEach({ addSubview($0) })
```
Comment (LucianoPAlmeida) [inline Sources/Extensions/UIKit/UIViewExtensions.swift]: Here too :))

File: Sources/Extensions/UIKit/UITextFieldExtensions.swift [hunk 3]
```diff
@@ -152,9 +152,9 @@ public extension UITextField {
     public func addPaddingLeftIcon(_ image: UIImage, padding: CGFloat) {
         let imageView = UIImageView(image: image)
         imageView.contentMode = .center
-        self.leftView = imageView
-        self.leftView?.frame.size = CGSize(width: image.size.width + padding, height: image.size.height)
-        self.leftViewMode = UITextField.ViewMode.always
+        leftView = imageView
+        leftView?.frame.size = CGSize(width: image.size.width + padding, height: image.size.height)
+        leftViewMode = UITextField.ViewMode.always
```
Comment (guykogus) [inline Sources/Extensions/UIKit/UITextFieldExtensions.swift]: Can this be changed to `leftViewMode = .always`?

**Comment by guykogus** [comment]
Very happy to see this! Good stuff :) Just a few comments left here for you.

**Comment by staffler-xyz** [comment]
sure, changes from comments added :)

**Pull request merged**